### ISSUE TITLE
README.md?

### ISSUE THREAD

**Issue by arcreative** [jvoorhis/lucene_query#1]
Just a couple usage examples would be great!

**Comment by jvoorhis** [comment]
As you can tell from the commit history, this project has not been very active lately. I suggest you check out the rspec tests at https://github.com/jvoorhis/lucene_query/blob/master/examples/lucene_query.rb.

**Comment by arcreative** [comment]
Yeah, saw that, but those seem to be written for rspec and not use cases.  Was trying:

``` ruby
LuceneQuery.new(&lambda {
  Fuzzy('test')
})
```

but it didn't like that so much...
